rules/windows/defense_evasion_microsoft_defender_tampering_by_registry.toml

```
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/10/18"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when one or more features on Microsoft Defender is disabled
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Microsoft Defender Tampering by Registry"
```

```suggestion
name = "Microsoft Defender Tampering"
```
I would suggest to confirm if all other aproaches (WMI, Posh, Reg) will result in this event, but it is likely that yes.

Ref:
https://github.com/preludeorg/community/blob/80ec8edcbfb81816123a76f1309765bb87d03e92/ttps/defense-evasion/b007f6e8-4a87-4440-8888-29ceab047d9b.yml

---

It was not something to add, but to verify if it is generic or specific to one of the approaches, did you test it? @austinsonger